### ISSUE TITLE
Error when training .

### ISSUE THREAD

**Issue by ThorPham** [universea/Baidu_Lane_Segmentation#3]
I have gtx 1080TI and installed paddlepaddle-gpu . When i training have error .How can i fix it

  train(model)
  File "train.py", line 160, in train
    exe.run(fluid.default_startup_program())
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/executor.py", line 650, in run
    use_program_cache=use_program_cache)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/executor.py", line 748, in _run
    exe.run(program.desc, scope, 0, True, True, fetch_var_name)
paddle.fluid.core_avx.EnforceNotMet: Invoke operator fill_constant error.
Python Callstacks: 
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/framework.py", line 1819, in _prepend_op
    attrs=kwargs.get("attrs", None))
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/initializer.py", line 186, in __call__
    stop_gradient=True)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/framework.py", line 1602, in create_var
    kwargs['initializer'](var, self)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/layer_helper_base.py", line 383, in set_variable_initializer
    initializer=initializer)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/layers/tensor.py", line 139, in create_global_var
    value=float(value), force_cpu=force_cpu))
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/optimizer.py", line 226, in _create_global_learning_rate
    persistable=True)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/optimizer.py", line 365, in _create_optimization_pass
    self._create_global_learning_rate()
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/optimizer.py", line 531, in apply_gradients
    optimize_ops = self._create_optimization_pass(params_grads)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/optimizer.py", line 561, in apply_optimize
    optimize_ops = self.apply_gradients(params_grads)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/optimizer.py", line 600, in minimize
    loss, startup_program=startup_program, params_grads=params_grads)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/dygraph/base.py", line 88, in __impl__
    return func(*args, **kwargs)
  File "/home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/paddle/fluid/wrapped_decorator.py", line 25, in __impl__
    return wrapped_func(*args, **kwargs)
  File "</home/thorpham/anaconda3/envs/segment/lib/python3.5/site-packages/decorator.py:decorator-gen-20>", line 2, in minimize
  File "train.py", line 156, in train
    optimizer.minimize(loss)
  File "train.py", line 224, in <module>
    train(model)
C++ Callstacks: 
Cannot load cudnn shared library. Cannot invoke method cudnnGetVersion at [/paddle/paddle/fluid/platform/dynload/cudnn.cc:59]

**Comment by ThorPham** [comment]
I FIXED.
